### PR TITLE
added total trading volume for vela

### PR DESCRIPTION
**Pull request by zodiacasti**

### CODE REVIEW

**Comment by llamatester** [comment]
The vela adapter exports: 
        
 
 _______________________________________
    Dexs for 1/4/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 1/2/2023
    Timestamp: 1680393598
    Trade volume: 5156800586.730268
    └─ Methodology: Total trading volume
    
    
    
    
    Done in 3.75s.

File: dexs/vela/index.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const graphs = (graphUrls: ChainEndpoints) => {
+  return (chain: Chain) => {
+    return async (timestamp: number) => {
+      const graphRes = await request(graphUrls[chain], graphQuery);
+      const totalVolume = parseInt(graphRes.globalInfos[0].volume) / 1e30;
+
+      return {
+        timestamp,
+        tradeVolume: totalVolume.toString(),
```
Comment (dtmkeng) [inline dexs/vela/index.ts]: for total trade volume should set on field `totalVolume`
and could you and daily trade volume it should set on field `dailyVolume`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/vela/index.ts (hunk 1 above)
Comment (zodiacasti) [reply dexs/vela/index.ts]: done

**Comment by llamatester** [comment]
The vela adapter exports: 
        
 
 _______________________________________
    Dexs for 1/4/2023
    _______________________________________
    
    1680307200
    ARBITRUM 👇
    Backfill start time: 1/2/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1680393598
    Total volume: 5158255515.5242815
    Daily volume: 10754276.210359165
    
    
    
    
    Done in 4.69s.

**Comment by llamatester** [comment]
The vela adapter exports: 
        
 
 _______________________________________
    Dexs for 1/4/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 1/2/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1680393598
    Total volume: 5158371669.260231
    Daily volume: 10754276.210359165
    
    
    
    
    Done in 6.42s.

**Pull request merged**